charts/osm/values.yaml

```
@@ -12,7 +12,7 @@ osm:
     # -- Container image pull policy for control plane containers
     pullPolicy: IfNotPresent
     # -- Container image tag for control plane images
-    tag: "latest-main"
+    tag: "v1.1.0"
```

This should be blank so the digests take precedent by default.
```suggestion
    tag: ""
```